pygsheets/sheet.py

```diff
@@ -51,12 +51,34 @@ def set_batch_mode(self, mode):
         self.batch_mode = mode
         self.batched_requests = dict()
 
-    def run_batch(self):
-        for ss, req in self.batched_requests.items():
+    def run_batch(self, verbose=False, _return_list_if_single=False):
```

Comment:
please rename `_return_list_if_single` to `return_list_if_single` for consistency,  as we don't use `_` in naming for params.